File: sdkwork-app-sdk-csharp/Models/DeviceRegisterForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class DeviceRegisterForm
    {
        public string? DeviceType { get; set; }
        public string? DeviceId { get; set; }
        public string? PushToken { get; set; }
        public string? OsVersion { get; set; }
        public string? AppVersion { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/device-register-form.ts
/** 设备注册表单 */
export interface DeviceRegisterForm {
  /** 设备类型 */
  deviceType: string;
  /** 设备标识符 */
  deviceId: string;
  /** 推送令牌 */
  pushToken: string;
  /** 操作系统版本 */
  osVersion?: string;
  /** 应用版本 */
  appVersion?: string;
}
